client/components/auth/auth.service.js

```diff
@@ -152,6 +156,7 @@ function AuthService($location, $http, $cookieStore, $q, appConfig, Util, User)
      */
     getCurrentUser(callback) {
       if (arguments.length === 0) {
+        smallgroup._id ? currentUser.smallgroup = true : currentUser.smallgroup = false;
```

Comment:
just found out that "null" actually wouldn't work for === or !== it has to be "undefined", interesting